frontend/app/professor/simulation-builder/page.tsx

```
@@ -1981,258 +2234,273 @@ return (
+                     ? 'border-blue-500 bg-blue-50 scale-105'
+                     : uploadedFile
+                     ? 'border-green-500 bg-green-50'
+                     : 'border-gray-300 bg-card hover:border-gray-400'
+                 }`}
+                 onDragOver={handleDragOver}
+                 onDragLeave={handleDragLeave}
+                 onDrop={handleDrop}
+                 onClick={() => fileInputRef.current?.click()}
+               >
+                 {uploadedFile ? (
+                   <span className="flex flex-col items-center">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)